hacking/shippable/get_recent_coverage_runs.py

```diff
@@ -53,12 +53,24 @@ def get_coverage_runs():
 
 
 def pretty_coverage_runs(runs):
-    for run in sorted(runs, key=lambda x: x['endedAt']):
+    ended = []
+    in_progress = []
+    for run in runs:
+        if run.get('endedAt'):
+            ended.append(run)
+        else:
+            in_progress.append(run)
+
+    for run in sorted(ended, key=lambda x: x['endedAt']):
         if run['statusCode'] == 30:
             print('🙂 [PASS] https://app.shippable.com/github/ansible/ansible/runs/%s (%s)' % (run['runNumber'], run['endedAt']))
         else:
             print('😢 [FAIL] https://app.shippable.com/github/ansible/ansible/runs/%s (%s)' % (run['runNumber'], run['endedAt']))
 
+    if in_progress:
+        print('The following runs are ongoing:')
+        for run in in_progress:
+            print('🤔 [FATE] https://app.shippable.com/github/ansible/ansible/runs/%s' % run['runNumber'])
```

Comment:
Why `FATE` ?

Reply:
Nobody knows if it'll pass or fail. It's up to fate! :)

Reply:
(Really, I just wanted a different 4 letter string to use there so it lined up with the ones above, and `FATE` amused me more than `UNKN` or similar.)